runtime/Cpp/runtime/src/atn/LL1Analyzer.cpp

```diff
@@ -31,16 +166,11 @@ std::vector<misc::IntervalSet> LL1Analyzer::getDecisionLookahead(ATNState *s) co
 
   look.resize(s->transitions.size()); // Fills all interval sets with defaults.
   for (size_t alt = 0; alt < s->transitions.size(); alt++) {
-    bool seeThruPreds = false; // fail to get lookahead upon pred
-
-    ATNConfig::Set lookBusy;
-    antlrcpp::BitSet callRuleStack;
-    LOOK(s->transitions[alt]->target, nullptr, PredictionContext::EMPTY,
-          look[alt], lookBusy, callRuleStack, seeThruPreds, false);
-
+    LL1AnalyzerImpl impl(_atn, look[alt], false, false);
+    impl.LOOK(s->transitions[alt]->target, nullptr, PredictionContext::EMPTY);
     // Wipe out lookahead for this alternative if we found nothing
     // or we had a predicate when we !seeThruPreds
-    if (look[alt].size() == 0 || look[alt].contains(HIT_PRED)) {
+    if (look[alt].size() == 0 || look[alt].contains(Token::INVALID_TYPE)) {
```

Comment:
This probably doesn't change the value which is used to indicate the tested situation, but it changes semantic. Without looking up what HIT_PRED is, I can guess it means a predicate was hit. An invalid token type is something completely different. So why changing that at all?

Reply:
I had switched it to make compilation work as HIT_PRED was defined in LL1Analyzer. I should have just qualified with as `LL1Analyzer::HIT_PRED`.

Switched it back, was being lazy.